pages/[...slug].tsx

```diff
@@ -405,15 +411,18 @@ export const GenericPage: React.FC<GenericPageProps> = ({
     </OnThisPage>
   )
 
-  const contentBanner = (
+  const contentBanner = !hideContentBanner ? (
```

Comment:
It can be more readable if this is written as `hideContentBanner ? null : <ContentBanner ...`